classes/mcPy/McPy.py

```diff
@@ -20,17 +23,15 @@ def get_available_core():
 
 
 def _launch(parser: Parser):
-    try:
-        # noinspection PyUnresolvedReferences
-        from blackfire import probe  # Profiler: https://blackfire.io free with the Git Student Package
-    except ImportError:
-        BLACKFIRE_ENABLED = False
-        logging.info("Blackfire not installed: passing")
-    else:
+    if (config['use_blackfire'] == True):
         BLACKFIRE_ENABLED = True
+        from blackfire import probe
```

Comment:
I would retain the try-except clause just for safety (never underestimate the end user)